<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Update</class>
 <widget class="QDialog" name="Dialog_Update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmware Update</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rrcc.qrc">
    <normaloff>:/png/png/parcel.png</normaloff>:/png/png/parcel.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_file">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Filename</string>
        </property>
        <property name="text">
         <string notr="true">?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_upload">
        <property name="toolTip">
         <string>Upload Status</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_install">
        <property name="toolTip">
         <string>Install Status</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_bytes">
          <property name="toolTip">
           <string>Transfered Bytes</string>
          </property>
          <property name="text">
           <string notr="true">0 / 0 Byte</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_time">
          <property name="toolTip">
           <string>Elapsed Time</string>
          </property>
          <property name="text">
           <string notr="true">00h:00m:00s</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>Update takes up to 10min!
Be patient and listen to voice output...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rrcc.qrc"/>
 </resources>
 <connections/>
</ui>
